src/main/java/mate/academy/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,46 @@
+package mate.academy.dao.impl;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import mate.academy.dao.OrderDao;
+import mate.academy.lb.Dao;
+import mate.academy.model.Order;
+import mate.academy.model.ShoppingCart;
+import mate.academy.storage.Storage;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        Order order = new Order(shoppingCart.getUserId());
```

Comment:
move this logic to the service layer 
add products to order 
clear shopping cart